src/components/com_tjdashboard/administrator/tables/dashboards.php

```diff
@@ -92,4 +92,30 @@ public function check()
 
 		return parent::check();
 	}
+
+	/**
+	 * Method to delete a dashboard.
+	 * 
+	 * @param   mixed  $pk  Primary key value to delete. Optional
+	 * 
+	 * @return  boolean  True on success.
+	 *
+	 * @since   1.0.0
+	 */
+	public function delete($pk = null)
+	{
+		JSession::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
+
+		try
+		{
+			$this->load($pk);
+			$result = parent::delete($pk);
```

Comment:
@shitalkakade24 Update this as per discussion

Reply:
done